### ISSUE TITLE
SetOrder is missing sendRequest argument

### ISSUE THREAD

**Issue by carrotinside** [matfish2/vue-tables-2#940]
- Vue.js version: 2.6.12
- vue-tables-2 version: 2.2.1
- consumed using: webpack

I can't find third argument 'sendRequest' of 'SetOrder' method. I would like to use it to avoid duplicated load.
Thank you

![image](https://user-images.githubusercontent.com/56627298/112956345-a78e7e00-9140-11eb-8a0e-6eb371590a5a.png)

![image](https://user-images.githubusercontent.com/56627298/112955138-66e23500-913f-11eb-921d-34b0fc33e2eb.png)

**Comment by matfish2** [comment]
You can use `setRequestParams` to set multiple params with a single call: https://matanya.gitbook.io/vue-tables-2/server-table/setting-multiple-request-parameters

**Comment by carrotinside** [comment]
Yes, but it calls requestFunction for every param.
I'm looking for replacement of `table.initOrderBy();` I used in previous version.

**Comment by matfish2** [comment]
> Yes, but it calls requestFunction for every param.

It doesn't, or at least it shouldn't... The whole idea is to set **multiple** parameters and send a **single** request.

If you are looking to set initial order use the `orderBy` option as documented in the options API (https://matanya.gitbook.io/vue-tables-2/options-api)

**Comment by matfish2** [comment]
Unable to reproduce.

Calling `setRequestParams`:

![2021-03-30 (2)](https://user-images.githubusercontent.com/1510460/113052816-42727100-91b0-11eb-9f2e-ea5d8b48bb30.png)

Only one request sent with all parameters:

![2021-03-30](https://user-images.githubusercontent.com/1510460/113052812-40a8ad80-91b0-11eb-83b9-028841762c40.png)

**Comment by carrotinside** [comment]
I see it like this (server-table):

![image](https://user-images.githubusercontent.com/56627298/113095145-3eb40e00-91f3-11eb-8056-fb56247d528f.png)

4 params:
![image](https://user-images.githubusercontent.com/56627298/113094663-5dfe6b80-91f2-11eb-8336-c7c8b09f84f2.png)

![image](https://user-images.githubusercontent.com/56627298/113094606-43c48d80-91f2-11eb-9efd-b595968158f0.png)

2 params:
![image](https://user-images.githubusercontent.com/56627298/113094849-b897c780-91f2-11eb-996d-b75ef706799e.png)

![image](https://user-images.githubusercontent.com/56627298/113094799-9f8f1680-91f2-11eb-953c-168ba0673aa2.png)

**Comment by matfish2** [comment]
Tried with `requestFunction`, still unable to reproduce. as for the third argument of `setOrder` it is set to `true` by default, so the *compiled* version removes it from the arguments, and checks for it in the body of the function:
```
  var sendRequest = arguments.length > 2 && arguments[2] !== undefined ? arguments[2] : true;
```
So if you pass `false` as the third argument the request should not be sent.

**Comment by carrotinside** [comment]
This is simplified `requestFunction`, but with same issue:
![image](https://user-images.githubusercontent.com/56627298/113997725-b53ab680-9858-11eb-97c2-67a9fb69dd23.png)

When I set params only for `order`, `requestFunction` calls twice:
![image](https://user-images.githubusercontent.com/56627298/113998103-077bd780-9859-11eb-94a5-d16da730e355.png)
![image](https://user-images.githubusercontent.com/56627298/113998265-32662b80-9859-11eb-8869-1b9957141f1b.png)

And when only for `customFilters`, `requestFunction` calls once:
![image](https://user-images.githubusercontent.com/56627298/113998763-aa345600-9859-11eb-8a6e-faa2140d51c9.png)
![image](https://user-images.githubusercontent.com/56627298/113999368-452d3000-985a-11eb-8b9b-884b61501b42.png)

**Comment by pestev** [comment]
I have similar problem. Using server tables with Vuex.
`this.$refs.invoicesTable.setRequestParams( { ... }, false );`
Every argument param call server. Looks like vuex mutations is making this calls.
All mutations has this part of code. Missing there condition if I want fire getData() or not.
if (self.source == 'server') self.getData()

**Comment by matfish2** [comment]
Fixed in 2.3.0
